README.md

```diff
@@ -17,9 +17,24 @@ The configuration `config.json` contains following properties in `parameters` ke
 - `db` - object (required): Connection settings
     - `host` - string (required): IP address or hostname of Apache Hive DB server
     - `port` - integer (required): Server port (default port is `10000`)
-    - `user` - string (required): User with correct access rights
-    - `#password` - string (required): Password for given `user`
-    - `database` - string (required): Database to connect to
+    - `database` - string (required): Database to connect to.
+    - `authType` - enum (optional): Type of the authentication.
+        - `password` (default) - user/password LDAP authentication.
+        - `kerberos` - Kerberos authentication.
+    - `user` - string (required if `authType = password`): User with correct access rights
+    - `#password` - string (required if `authType = password`): Password for given `user`
+    - `kerberos` - object (required if `authType = kerberos`)
+        - `principal` - string (required): Name of the Kerberos principal - used for the `kinit`.
+        - `config` - string (required): Content of the `krb5.conf` file.
+        - `#keytab` - string (required): `base64` encoded content of the `*.keytab` file.
```

Comment:
Keytab je nieco ako heslo/certifikat. Je to binarny subor, preto musi byt v configu base64 encoded.